user_tests.py

```diff
@@ -1,15 +1,12 @@
-import user_category
+from user_category import user_type
```

Comment:
Why not import both number and user_type from user_category forexample
```python
from user_category import user_type, number
```

Reply:
@dennisja , I think your help today through TDD will be of great value coz I have questions on testing number function with input

Reply:
Hope so, will be waiting to see and talk to you then

Reply:
Alright thanks